tests/functional/test_bad_url.py

```diff
@@ -0,0 +1,24 @@
+# test the error message returned by pip when
+# a bad "file:" URL is passed to it.
+
+from typing import Any, Tuple
+
+
+def get_url_error_message(script: Any, fake_file: str) -> Tuple[str, str, int]:
```

Comment:
Even when I know what `script` is, I didn't know how to annotate it at all, so I used `Any`. Is there something else for doing that?

Reply:
Good enough since this is just for testing purposes. If it fails, some tests will probably fail.